src/test/resources/munit/body/form/multipart-form-array/library/sample-types.raml

```diff
@@ -0,0 +1,33 @@
+#%RAML 1.0 Library
+
+usage: |
+    Sample Types
+
+types:
+  
+  IndexValue:
+    displayName: Sample Index Value
+    type: object
+    xml:
+      name: index
+    properties:
+      name:
+        xml:
+          attribute: true
+        type: string
+      value:
+        xml:
+          attribute: true
+        type: string
+
+  Metadata:
+    displayName: Sample MetaData
+    xml:

```

Comment:
I see the example as a JSON, Is this intended to be an xml? Wondering if the xml node makes a difference for the validation

Reply:
Yes you're right I need to remove that an other this at the api/test that are related to the customer. Will do.